source/adios2/helper/adiosSystem.cpp

```diff
@@ -50,7 +50,6 @@ std::string LocalTimeDate() noexcept
 {
     std::time_t now =
         std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
-
```

Comment:
There is still a line removal here. Please squash the commits and make sure `adiosSystem.cpp` is unchanged.